src/main/java/wansbot/tasks/Deadlined.java

```diff
@@ -0,0 +1,47 @@
+package wansbot.tasks;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class which provides deadline functions to WansBot.
+ */
+public class Deadlined extends Task {
+    private LocalDate deadline;
+
+    /**
+     * Constructor that takes the name and date and assigns it to the Deadlined.
+     */
+    public Deadlined(String name, LocalDate deadline) {
+        super(name);
+        this.deadline = deadline;
+    }
+
+    /**
+     * If Deadlined's date is equal to date, then returns true. Returns false otherwise.
+     *
+     * @param date is a LocalDate input by user as part of filtering tasks.
+     * @return Boolean which tells user if the Deadline is by date.
+     */
+    public boolean isOnDate(LocalDate date) {
```

Comment:
Good job using proper naming for booleans!

Reply:
Thanks bro!